backend/routes/orgUsers.js

```diff
@@ -1,8 +1,10 @@
 const express = require("express");
 const router = express.Router();
-const {loginOrgUser, createOrgUser} = require("../controllers/orgUsersController");
+const {loginOrgUser, createOrgUser, findOrgUserById, getAllOrgUsers} = require("../controllers/orgUsersController");
 
 router.post("/login", loginOrgUser);
 router.post("/signup", createOrgUser);
+router.get("/:org_user_id", findOrgUserById);
```

Comment:
On the backend because I worked on the comments, I have used `orgUser_id:` and `indUser_id:` is it ok if we use the same to keep consistency

Reply:
if we are using underscores than we should use underscores, if we are using camelcase we should use camelcase, i dont think a mix of both is good practise or natural, I should of had a better look when doing your pull request, apologies, can change if its really an issue but a quick find and replace should solve everything

Reply:
Sorry for the confusing, I was using camelcase but with mongodb id, it's written as _id  hence the mix case but find with you changing it to all one format if you want